src/index.d.ts

```diff
@@ -1,13 +1,18 @@
-declare module 'react-tater' {
-  import React from 'react';
+import { ReactElement } from 'react';
 
-  interface ReactTaterProps {
-    name?: string
-    space?: number
-    children: React.Element
-  }
-
-  const Tater: (props: ReactTaterProps) => ReactElement<ReactTaterProps>
+interface TaterOptions {
+  name?: string;
+  space?: number;
+}
 
-  export default Tater
+interface TaterProps {
+  options?: TaterOptions;
+  children: ReactElement;
 }
+
+declare const Tater: (props: TaterProps) => JSX.Element;
```

Comment:
Why is the return type `JSX.Element` instead of `ReactElement`?

Reply:
Ah, I see https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement